### ISSUE TITLE
Tests failing with unexpected result

### ISSUE THREAD

**Issue by richardlford** [dotnet/coreclr#2414]
The following tests fail to return the expected return code, thus return
false rather than true from the test driver. These tests
did not product any other output to indicate the cause of failure.
This was for a debug win64 run.

```
JIT\IL_Conformance\Old\Conformance_Base\rem_r4\rem_r4.cmd
JIT\jit64\regress\ndpw\21220\b21220\b21220.cmd
JIT\jit64\regress\vsw\102754\test1\test1.cmd
JIT\jit64\regress\vsw\102754\test2\test2.cmd
JIT\Methodical\switch\switch1\switch1.cmd
JIT\Methodical\switch\switch10\switch10.cmd
JIT\Methodical\switch\switch11\switch11.cmd
JIT\Methodical\switch\switch2\switch2.cmd
JIT\Methodical\switch\switch3\switch3.cmd
JIT\Methodical\switch\switch4\switch4.cmd
JIT\Methodical\switch\switch5\switch5.cmd
JIT\Methodical\switch\switch6\switch6.cmd
JIT\Methodical\switch\switch7\switch7.cmd
JIT\Methodical\switch\switch8\switch8.cmd
JIT\Methodical\switch\switch9\switch9.cmd
JIT\Methodical\tailcall\_il_dbgrecurse_ep_void\_il_dbgrecurse_ep_void.cmd
JIT\Methodical\tailcall\_il_dbgtest_void\_il_dbgtest_void.cmd
JIT\Methodical\tailcall\_il_relrecurse_ep_void\_il_relrecurse_ep_void.cmd
JIT\Methodical\tailcall\_il_reltest_void\_il_reltest_void.cmd
JIT\Regression\CLR-x86-JIT\V1-M12-Beta2\b65423\b65423\b65423.cmd
```

**Comment by hoyMS** [comment]
I didn't see those failures in a recent x64 windows debug run.

**Comment by AndyAyersMS** [comment]
@hoyMS are you going to enable these tests then? 

You'll need to edit issues.targets.

**Comment by RussKeldorph** [comment]
Not sure what @hoyMS was observing.  These are still failing for me.

**Comment by RussKeldorph** [comment]
The following were broken out of this issue into separate issues:

#4850
```
JIT\IL_Conformance\Old\Conformance_Base\conv_ovf_i8_i\conv_ovf_i8_i.cmd
```
#4851
```
JIT\Directed\coverage\importer\Desktop\bleref_il_d\bleref_il_d.cmd
JIT\Directed\coverage\importer\Desktop\bleref_il_r\bleref_il_r.cmd
JIT\jit64\localloc\verify\verify01_dynamic\verify01_dynamic.cmd
JIT\jit64\localloc\verify\verify01_large\verify01_large.cmd
JIT\jit64\localloc\verify\verify01_small\verify01_small.cmd
```
#4853
```
JIT\Methodical\cctor\misc\tail_il_d\tail_il_d.cmd
JIT\Methodical\cctor\misc\tail_il_r\tail_il_r.cmd
JIT\Methodical\cctor\misc\threads3_il_d\threads3_il_d.cmd
JIT\Methodical\cctor\misc\threads3_il_r\threads3_il_r.cmd
JIT\Methodical\cctor\simple\precise3_il_d\precise3_il_d.cmd
JIT\Methodical\cctor\simple\precise3_il_r\precise3_il_r.cmd
```
#4855
```
JIT\Methodical\varargs\seh\fault_il_d\fault_il_d.cmd
JIT\Methodical\varargs\seh\fault_il_r\fault_il_r.cmd
```
#4849
```
JIT\Regression\CLR-x86-JIT\V1.2-M01\b08046\b08046\b08046.cmd
```

**Comment by RussKeldorph** [comment]
Fixed by #4845.
